avocado/core/runner.py

```diff
@@ -233,21 +228,13 @@ def _run_test(self, test_factory, queue):
         sys.stdout = output.LoggingFile(logger=logger_list_stdout)
         sys.stderr = output.LoggingFile(logger=logger_list_stderr)
 
-        try:
-            instance = loader.load_test(test_factory)
-            if instance.runner_queue is None:
-                instance.runner_queue = queue
-            runtime.CURRENT_TEST = instance
-            early_state = instance.get_state()
-            early_state['early_status'] = True
-            queue.put(early_state)
-        except Exception:
```

Comment:
Well this was here for a reason. When anything here fails, avocado hangs. It's true that we added 10s deadline, so maybe it's time to say good buy to this, but I'd prefer keep it here (and remove the `Exception` to catch really everything).